### ISSUE TITLE
Using Visual Studio Code-insider with Centos 7 crashes 

### ISSUE THREAD

**Issue by webzest** [microsoft/vscode#107175]
I installed a new Hyper-V Centos 7 for Data Science.  I installed Miconda3 and created a new environment with all the necessary packages.  Then, I installed Visual Studio Code-Insider.  So far so good, no errors.  then, I launched Code and select to open a Folder.  VSCode closed immediately and nothing happened after that.  I also tried tto Open a folder on a new VSCode install and the same behavior.  

I am not able to Save files either.  If I try to Create New File and then select Save or Save As, the same behavior is achieved.

So, After spending four hous installing a new OS for Data Science, the actual thing I need to use for data Science doesn't work.  I am disappointed.  

Any help would be appreciated...

**Comment by IllusionMH** [comment]
/needMoreInfo

Are you using snap package? 
Looks like duplicate of #100940 and workaround https://github.com/microsoft/vscode/issues/100940#issuecomment-678711777

**Comment by IllusionMH** [comment]
/needsMoreInfo

Are you using snap package? 
Looks like duplicate of #100940 and workaround https://github.com/microsoft/vscode/issues/100940#issuecomment-678711777

**Comment by webzest** [comment]
> /needsMoreInfo
> 
> Are you using snap package?
> Looks like duplicate of #100940 and workaround [#100940 (comment)](https://github.com/microsoft/vscode/issues/100940#issuecomment-678711777)

Yes, I followed the Visual Studio Code page: https://code.visualstudio.com/docs/setup/linux 
Which pointed me to use snap.  I did and it killed me.  They should remove this link and instructions because snap is a killer.

After removing all snapd installed apps, snapd itself, rebooted and then using the yum version from here: https://linuxize.com/post/how-to-install-visual-studio-code-on-centos-7/
I was able to use VSCode as intended.

Thank you much for the alert...
